### PR TITLE
[SimpleSound 2.1] Initial upload

### PR DESCRIPTION
**Pull request by lazy468**

### CODE REVIEW

File: SimpleSound/script.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Simple Sound",
+    "script": "simplesound.js",
+    "version": "2.1",
+    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.",
```
Comment (Anduh) [inline SimpleSound/script.json:5]:
```suggestion
    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions",
```
Link to Readme + show some commands makes it easier to get started.

**Review by Anduh — CHANGES_REQUESTED** [review]
The extra GPL license should probably be removed. By submitting the API to the repo, you're agreeing to the repo's [existing MIT license](https://github.com/Roll20/roll20-api-scripts#license).

Tnaks for submitting it to the API menu, will make it easier to use!

**Comment by lazy468** [comment]
Very good, I've removed the extra licensing information and committed your suggestion to description. Thanks!

**Review by Anduh — CHANGES_REQUESTED** [review]
Glad to help!

Few more things I noticed:
* Each API version needs to be in a folder, including the latest one. API authors just commonly leave a version of the latest version in the root folder so it's easier to find for those navigating the repo, but it's in no way necessary. 
* As this is the initial publishing of the API, IMO you can just simplify and only upload version `2.1`, given the trivial changes between `2.0` and `2.1`
* `2.0/Simplesounds.js` still have the GPL license in the comments

File: SimpleSound/script.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Simple Sound",
+    "script": "simplesound.js",
+    "version": "2.1",
+    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions",
```
Comment (Anduh) [inline SimpleSound/script.json:5]:
```suggestion
    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions\n\n[API:Simple Sound](https://app.roll20.net/forum/post/10593263/script-simple-sound)-forum thread",
```

Comment (Anduh) [inline SimpleSound/script.json:5]:
```suggestion
    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions\n\n[API:Simple Sound](https://app.roll20.net/forum/post/10593263/script-simple-sound)-forum thread",
```

Comment (Anduh) [inline SimpleSound/script.json:5]:
```suggestion
    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions\n\n[API:Simple Sound](https://app.roll20.net/forum/post/10593263/script-simple-sound)-forum thread",
```

Comment (Anduh) [inline SimpleSound/script.json:5]:
```suggestion
    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions\n\n[API:Simple Sound](https://app.roll20.net/forum/post/10593263/script-simple-sound)-forum thread",
```

**Review by NicholasR77 — APPROVED** [review]

**Pull request merged**